content/utilities/tiny-leaderboard/plugin.js

```diff
@@ -0,0 +1,171 @@
+// Tiny Leaderboard
+//
+// Shows a tiny leaderboard with a timer (until the round ends)
+// Shows the 5 people above and below you on the board
+// Updates every 60 seconds
+
+const updateTimeInSeconds = 60;
+const leaderboardRange = 5; // 5 people above and below you
+
+// ---------------------------------------------------------------
+
+// https://github.com/darkforest-eth/client/blob/19f2547dce33f3274afa86945e96fbfebc7693f5/src/Frontend/Views/Leaderboard.tsx#L119
+const roundEndTimestamp = '2021-07-07T21:00:00-07:00';
```

Comment:
We can do 
`df.endTimeSeconds`

Reply:
nice, i didnt know that existed, i can do a commit to use it instead

Reply:
ok i just tested it and:
`new Date(df.endTimeSeconds*1000)`
gave me:
`Mon Jan 31 2022 01:05:33 GMT+0100`
does it mean next round will end at 31 Jan next year? or is the number currently wrong?

Reply:
i made the commit now, i assume the number will be correct as soon as the next round starts, or the next round will be long :D

Reply:
phated said on discord:

> Pretty sure round end is when the smart contract stops accepting moves. Different from the actual next round

maybe this is not the round end time

Reply:
I have an issue in to the darkforest repo to try to upstream a new field maybe

Reply:
Probably dont need it for this pr but there is a roundend in the contract now. Not sure if its plumbed through to df.endTimeSeconds
https://github.com/darkforest-eth/eth/blob/cd1388ebbeee41a0db141ec2d32bf4abb2082c88/contracts/DarkForestScoringR3.sol#L18

Reply:
i dont know how to read this variable from the plugin, but i will try to figure it out until the next round